testbed/compositions/graphsync.toml

```diff
@@ -16,23 +16,20 @@ instances = { count = 2 }
 
 [groups.build]
 
-
-
-
 [groups.run]
 [groups.run.test_params]
-input_data = "files"
-data_dir = "../extra/test-datasets"
+input_data = "random"
+file_size = "2000000000"
```

Comment:
@hannahhoward This dies when I try to randomly generate and add a 4GB file in Graphsync Node. It works for other nodes though. Any idea why this is ?

Reply:
I think probably better to use predefined files here for repeatable tests.

Reply:
this will enable the ignite team to customize your benchmarks without overwritng other teams benchmarks.

Reply:
@hannahhoward

1. Added Badger support to this repo so we can test with large files.
2. Reverted all changes to composition files. Will use the naming scheme you mention so as to not step on toes going ahead.